### PR TITLE
Fix loading projects by hash on player example

### PR DESCRIPTION
**Pull request by rschamp**
### Resolves

_What Github issue does this resolve (please include link)?_

Resolves `/player.html#<project id>` not loading the project with id `<project id>`

### Proposed Changes

_Describe what this Pull Request does_
Pass on uncaptured props from Player component to GUI

### Reason for Changes

_Explain why these changes should be made_
Passing on the props down from the HashParserHOC allows the HOC to load the project specified by the hash and give this information to the GUI.

### Test Coverage

_Please show how you have added tests to cover your changes_
I modified the integration tests to gather "performance" logs, which include which requests are made by the browser. This allows us to make sure that the project is actually being requested in the examples.

### Browser Coverage
N/A

### CODE REVIEW

**Comment by gnarf** [comment]
Tested this branch and everything seems to work! Thanks!

File: src/playground/player.jsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ const Player = ({isPlayerOnly, onSeeInside}) => (
         <GUI
             enableCommunity
             isPlayerOnly={isPlayerOnly}
+            {...props}
```
Comment (gnarf) [inline src/playground/player.jsx]: is there a specific prop this was missing? Curious if you really want to kind of steal the params list from GUI here, or just list the ones you care about, or if it wouldn't make more sense to pass GUI in as a child?

Comment (rschamp) [reply src/playground/player.jsx]: Good idea, I updated to use an explicit prop.

**Comment by paulkaplan** [comment]
I made that change @kchadha asked for, but I mucked up pushing it so things got a bit wacky in the commit history, sorry about that

**Pull request merged**